data/store/test/data_repository.go

```diff
@@ -126,6 +124,16 @@ type GetDataSetOutput struct {
 	Error   error
 }
 
+type IsDataSetAutomatedInput struct {
```

Comment:
All of the `IsDataSetAutomated` functionality can be removed, right?

Reply:
yes, good catch.